### ISSUE TITLE
A11y: In Time Picker time settings the list of time zones is not scrollable with a keyboard

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#56236]
**What happened**:
Change time settings inside the Time Picker in a dashboard: Arrow up and down in the timezones list is not moving the cursor, making the bottom rest of the list hidden. Users navigating with keyboard can not see the full list, even though they can select options that are not visible.

**What you expected to happen**:
Arrow up and down in the timezones list should also be moving the cursor, so that when users are navigating with keyboard they can see the necessary part of the list by preselecting any timezone.

**How to reproduce it (as minimally and precisely as possible)**:
Open Time Picker in a dashboard. Go to the "Change time settings" and navigate to the list of timezones. Use the arrow Up/Down to go through the list.

https://user-images.githubusercontent.com/13227501/193760583-0d585c83-f4c6-465a-a147-e3798c47ebad.mov

**Anything else we need to know?**:

**Relevant WCAG Criteria**:
[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 9.3.0
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:
